sponsor.html

```diff
@@ -0,0 +1,35 @@
+<head>
+	<link href="./dir.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+	<link rel='shortcut icon' type='image/x-icon' href='favicon.png' />
+	<title>DIR 2019 ⚡ Lightning Talk Edition ⚡ November 29, 2019 ⚡ Amsterdam</title>
+</head>
+
+<body>
+
+<div id="content">
+	<div id="block">
+		<div class="logo">
+			<span class="pad-right">DIR</span>
+			<span class="img"><img src="img/lightning.svg" /></span>
+			<span class="pad-left">19</span>
+		</div>
+	</div>
+	<div id="block">
+		<p>The 18th Dutch-Belgian Information Retrieval Workshop</p>
+		<p>Lightning Talk Edition ⚡ November 29, 2019 ⚡ Amsterdam</p>
+	</div>
+  
+	<div id="block" class="text">
+		<h1>Call for sponsors</h1>
+    <p>At this year's DIR, We expect 75-100 participants, mainly researchers and students in information retrieval and related fields from the Netherlands and Belgium (and some from the UK and Germany).</p>
+    <p>Sponsoring DIR can be a good opportunity to gain more prominence among this audience as a company that works in search, AI and data science. In addition, there is an opportunity to network with students and researchers in this field.</p> 
+    <p>We offer three sponsor packages:</p>
+    <ul>
+      <li>🥈For € 500 (silver): Logo and link on website. Logo in the program. An information table and 1 registration.</li>
+      <li>🥇For € 1000 (gold): All the above. Plus your own banner in the room and 2 registrations.</li>
+      <li>💍For € 2500 (platinum): All the above. Plus drink named after company.</li>
+    </ul>
+	</div>
```

Comment:
Should we add contact info/call to action? 😅

Reply:
yes